openssl/src/ssl/mod.rs

```diff
@@ -1227,7 +1227,7 @@ impl SslContextBuilder {
     ///
     /// [`SSL_CTX_set_psk_client_callback`]: https://www.openssl.org/docs/man1.0.2/ssl/SSL_CTX_set_psk_client_callback.html
     #[cfg(not(osslconf = "OPENSSL_NO_PSK"))]
-    pub fn set_psk_callback<F>(&mut self, callback: F)
+    pub fn set_psk_client_callback<F>(&mut self, callback: F)
```

Comment:
We can't just rename this since it'd break backwards compatibilty. Can you add a deprecated method with the old name? Something like this: https://github.com/sfackler/rust-openssl/blob/master/openssl/src/x509/mod.rs#L474-L477

Reply:
Should I leave the version as 0.10.9? If not what should I bump it to?

Reply:
You can use 0.10.10.